pkgs/by-name/gl/globalprotect-openconnect_2/gui.nix

```diff
@@ -0,0 +1,70 @@
+{
+  fetchurl,
+  stdenv,
+  zstd,
+  webkitgtk_4_0,
+  libappindicator,
+  gtk3,
+  cairo,
+  gdk-pixbuf,
+  libsoup,
+  glib,
+  autoPatchelfHook,
+  version,
+}:
+let
+  remoteHashes = {
+    "x86_64-linux" = {
+      sha256 = "sha256-wSoBvHkLbf9aaFWBJMfm7Angd/PwKzNH+yqvYxhjmxY=";
+      arch-name = "x86_64";
+    };
+    "aarch64-linux" = {
+      sha256 = "sha256-Gata5OVlckfcy/F+UBe2FW7OwB47efw69lMAAU8Q4JQ=";
+      arch-name = "aarch64";
+    };
+  };
+
+  remotes = builtins.mapAttrs (
+    _:
+    { sha256, arch-name }:
+    fetchurl {
+      inherit sha256;
+      url = "https://github.com/yuezk/GlobalProtect-openconnect/releases/download/v${version}/globalprotect-openconnect-${version}-1-${arch-name}.pkg.tar.zst";
+    }
+  ) remoteHashes;
+
+in
+stdenv.mkDerivation {
+  name = "gpgui";
+  inherit version;
+  src = remotes.${stdenv.hostPlatform.system};
```

Comment:
This should have a or throw unsupported system to not generate a syntax error on other platforms.